pkg/build/builder/util.go

```diff
@@ -85,90 +80,6 @@ func addBuildLabels(labels map[string]string, build *buildapiv1.Build) {
 	labels[builderutil.DefaultDockerLabelNamespace+"build.namespace"] = build.Namespace
 }
 
-// readInt64 reads a file containing a 64 bit integer value
-// and returns the value as an int64.  If the file contains
-// a value larger than an int64, it returns MaxInt64,
-// if the value is smaller than an int64, it returns MinInt64.
-func readInt64(filePath string) (int64, error) {
-	data, err := ioutil.ReadFile(filePath)
-	if err != nil {
-		return -1, err
-	}
-	s := strings.TrimSpace(string(data))
-	val, err := strconv.ParseInt(s, 10, 64)
-	// overflow errors are ok, we'll get return a math.MaxInt64 value which is more
-	// than enough anyway.  For underflow we'll return MinInt64 and the error.
-	if err != nil && err.(*strconv.NumError).Err == strconv.ErrRange {
-		if s[0] == '-' {
-			return math.MinInt64, err
-		}
-		return math.MaxInt64, nil
-	} else if err != nil {
-		return -1, err
-	}
-	return val, nil
-}
-
-// readNetClsCGroup parses /proc/self/cgroup in order to determine the container id that can be used
-// the network namespace that this process is running on, it returns the cgroup and container type
-// (docker vs crio).
-func readNetClsCGroup(reader io.Reader) (string, string) {
```

Comment:
@bparees these are now the concern of buildah (or completely irrelevant)?

Reply:
irrelevant.  Since the container is running inside the pod, it inherits the network cgroup as i understand it.  We needed this previously because the container ran outside the pod, so it picked up the host's network.